src/credentials/CHIPCert.h

```diff
@@ -182,6 +182,8 @@ enum
     kNullCertTime = 0
 };
 
+#define RDN_NUM 2
```

Comment:
Needs to be 3 per current spec, 5 after https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/2263